css/style.css

```diff
@@ -610,6 +1243,180 @@
   .arrow {
     padding-left: 2px;
   }
+
+  .info__title {
+    font-size: 68px;
+    text-align: right;
+    padding: 0 53px 0 0;
+    margin-top: 30px;
+  }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 899px;
+    top: 161px;
+  }
+
+  .wrap-information {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    background-position-y: top;
+    height: 995px;
+    margin-top: 20px;
+  }
+
+  .red-first {
+    max-width: 899px;
+    height: 487px;
+    float: none;
+    padding: 63px 47px 0 32px;
+  }
+
+  .redrow {
+    flex-direction: row;
+    display: flex;
+    height: 300px;
+    padding-bottom: 3px;
+  }
+
+  .description {
+    padding-right: 86px;
+  }
+
+  .description__title {
+    padding-bottom: 15px;
+  }
+
+  .icons {
+    padding: 5px 62px 0 0;
+  }
+
+  .description__screen {
+    padding-bottom: 28px;
+  }
+
+  .img-inch {
+    width: 36px;
+    height: 52px;
+  }
+
+  .data__figure {
+    font-size: 32px;
+    font-weight: 500;
+  }
+
+  .data__display {
+    font-size: 14px;
+  }
+
+  .description__wifi {
+    padding-bottom: 32px;
+  }
+
+  .img-wifi {
+    padding-right: 20px;
+    float: left;
+  }
+
+  .up-date {
+    white-space: nowrap;
+  }
+
+  .description__bag {
+    padding-bottom: 0;
+  }
+
+  .img-cap {
+    width: 36px;
+    height: 50px;
+    padding-top: 6px;
+  }
+
+  .data__space {
+    font-size: 12px;
+  }
+
+  .info-order {
+    display: flex;
+    justify-content: center;
+    padding: 17px 0 0 0;
+  }
+
+  .info-order__img {
+    width: 31px;
+    height: 31px;
+    padding-right: 14px;
+  }
+
+  .info-order__text {
+    font-size: 15px;
+    padding-right: 48px;
+  }
+
+  .wrap {
+    display: flex;
+    justify-content: flex-end;
+    flex-direction: column;
+    background-position: -24px 79px;
+    height: 1030px;
+  }
+
+  .details {
+    max-width: 899px;
+    height: 487px;
+    padding: 66px 47px 0 14px;
+    margin-top: 50px;
+  }
+
+  .redrow-scnd {
+    flex-direction: row;
+    display: flex;
+    padding-left: 10px;
+  }
+
+  .description-car {
+    padding-left: 13px;
+  }
+
+  .title {
+    padding-top: 0;
+  }
+
+  .description__title-most {
+    font-size: 30px;
+  }
+
+  .model-s {
+    font-size: 16px;
+    padding-right: 43px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)